### PR TITLE
Header/sidebar sizing

### PR DESCRIPTION
**Pull request by maykar**
## Proposed changes

* Shrinks header height and sidebar width down to 56px from the current 64px
* Makes all headers conform to `--header-height`
* Fixes alignment of sidebar header text

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Checklist

- [x] The code change is tested and works locally.
- [x] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/ha-sidebar.ts [hunk 1]
```diff
@@ -224,6 +224,11 @@ class HaSidebar extends LitElement {
       }
     }
 
+    const sidebarTitle =
```
Comment (balloob) [inline src/components/ha-sidebar.ts:227]: Please remove this change from this PR. This is removing all Home Assistant branding and that's something that needs to be discussed first.

**Comment by balloob** [comment]
Just a note that not every feature request needs to be implemented. We need to make sure that feature requests fit within our vision for what the frontend is, and what it is not.

**Comment by Mariusthvdb** [comment]
where can we discuss that Paulus? 
as it is, I can feel for the branding issue, hope you also see the user need for essential feedback on the system.

Wouldn't you agree the Frontend is for returning Essential feedback to the user, and much less so a branding vehicle

Especially so, since with the default (and not optionable) new compact header, the user is taken a spot where this resided before unfortunately.

**Comment by balloob** [comment]
Not interested in discussing that. Either use the custom JS that Maykar made or add a markdown card with the title.

**Comment by maykar** [comment]
Just in case other's are looking for it here, the custom js can be found here:

https://github.com/maykar/custom-sidebar-title

File: src/dialogs/notifications/notification-drawer.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ export class HuiNotificationDrawer extends EventsMixin(
         color: var(--primary-text-color);
         border-bottom: 1px solid var(--divider-color);
         background-color: var(--primary-background-color);
-        min-height: 64px;
+        height: calc(var(--header-height) - 1px);
```
Comment (zsarnett) [inline src/dialogs/notifications/notification-drawer.js:28]:
```suggestion
        height: var(--header-height);
        box-sizing: border-box;
```
?

Comment (maykar) [reply src/dialogs/notifications/notification-drawer.js:28]: border-box doesn't work in this particular situation, unless we adjust some other styling as well. With your changes above:

image

Comment (maykar) [reply src/dialogs/notifications/notification-drawer.js:28]: We could use box-sizing if we remove the width calculation on the same element `width: calc(100% - 32px);`

**Comment by bramkragten** [comment]
The header of `hass-loading-screen` is not adjusted yet

**Comment by maykar** [comment]
That rebase caused a good bit of the commits in this PR to be lost, will try to fix up when I get the chance. In it's current state this PR is incomplete.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**